### PR TITLE
CLIENTS: Get Byte Array

### PR DESCRIPTION
**Pull request by ElbekDeveloper**

### CODE REVIEW

File: RESTFulSense/Clients/RESTFulApiClient.cs [hunk 1]
```diff
@@ -429,5 +439,13 @@ public async ValueTask DeleteContentAsync(string relativeUrl)
                 ? JsonConvert.DeserializeObject<T>(responseString)
                 : await deserializationFunction(responseString);
         }
+
+        private async static ValueTask<byte[]> ValidateAndReturnInvalidResponse(
+            HttpResponseMessage httpResponseMessage)
+        {
+            await ValidationService.ValidateHttpResponseAsync(httpResponseMessage);
+
+            return null;
```
Comment (hassanhabib) [inline RESTFulSense/Clients/RESTFulApiClient.cs:448]: This looks like a hack to fit into the pattern-matching. Just do `if .. else`

Comment (ElbekDeveloper) [reply RESTFulSense/Clients/RESTFulApiClient.cs:451]: Resolved

File: RESTFulSense/Clients/RESTFulApiClient.cs [hunk 2]
```diff
@@ -429,5 +442,13 @@ public async ValueTask DeleteContentAsync(string relativeUrl)
                 ? JsonConvert.DeserializeObject<T>(responseString)
                 : await deserializationFunction(responseString);
         }
+
+        private async static ValueTask<byte[]> ValidateAndReturnInvalidResponse(
```
Comment (hassanhabib) [inline RESTFulSense/Clients/RESTFulApiClient.cs:446]: Remove this method - no longer needed.

Comment (ElbekDeveloper) [reply RESTFulSense/Clients/RESTFulApiClient.cs]: done

**Pull request merged**